ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -111,7 +113,8 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      tapsOnAppVersion: 0
+      tapsOnAppVersion: 0,
+      deviceUniqueId: DeviceInfo.getUniqueId()
```

Comment:
Why do you place this info into the screen state?
I don't find any problems about it but since it never changes (no re-draw needed for this value) I guess another option could be using this value directly without place it into the state.

thoughts?

Reply:
I thought it better to persist it rather than calling `Device.getUniqueId()` every render. 
But I don't think it really affects performance, so... no strong opinions here 😄 

We may ask for a third opinion to break the tie?

Reply:
I'm agree with you. I don't think about the cost of calling the native side.
Good point!